src/ProjectService.Business/Commands/Project/GetProjectCommand.cs

```diff
@@ -423,7 +434,26 @@ public async Task<OperationResultResponse<ProjectResponse>> ExecuteAsync(GetProj
         department = (await GetDepartmentAsync(dbProject.Id, null, response.Errors))?.FirstOrDefault();
       }
 
-      List<Guid> files = dbProject.Files.Select(x => x.FileId).ToList();
+      AccessType accessType = AccessType.Public;
+
+      DbProjectUser user = null;
+      List<DbProjectUser> users = await _userRepository.GetAsync(new List<Guid>() { _httpContextAccessor.HttpContext.GetUserId() });
```

Comment:
add FirstOrDeafult() in the end

Reply:
if ( dbProjectUser is not null)
{
      accessType = dbProjectUser.Role == ProjectRoleType.Manager? AccessType.Manager: AccessType.Team;

Reply:
done